src/routes/EResourceEditRoute.js

```diff
@@ -68,7 +83,7 @@ class EResourceEditRoute extends React.Component {
     this.props.history.push(`${urls.eresourceView(match.params.id)}${location.search}`);
   }
 
-  handleSubmit = (pci) => {
+  handlePCISubmit = (pci) => {
```

Comment:
Lets not have 2 different `handleSubmit` functions here. As if we need to extend this functionality to another new `eresource` we would need another function then. Lets have one `handleSubmit` that differentiates between a `pci` and a `title` submit, based on the `eresourceClass`